autokeras/engine/pipeline.py

```diff
@@ -0,0 +1,19 @@
+import kerastuner
+
+from autokeras.engine import serializable
+
+
+class Pipeline(kerastuner.HyperModel, serializable.Serializable):
+    """Input data pipeline search space.
+
+    This class defines the search space for input data pipelines. A pipeline
+    transoforms the dataset using tensorflow.data operations.
```

Comment:
We are trying to use this function:
https://www.tensorflow.org/api_docs/python/tf/data/Dataset#window
I don't think it can be applied to np.ndarray or EagerTensor.
Or is there an alternative solution?